code/modules/atmospherics/machinery/portable/portable_atmospherics.dm

```diff
@@ -190,3 +192,9 @@
 	investigate_log("was smacked with \a [item] by [key_name(user)].", INVESTIGATE_ATMOS)
 	add_fingerprint(user)
 	return ..()
+
+/// Holding tanks can get to zero integrity and be destroyed without other warnings due to pressure change. 
+/// This checks for that case and removes our reference to it.
+/obj/machinery/portable_atmospherics/proc/unregister_holding()
```

Comment:
Needs SIGNAL_HANDLER

Reply:
I'd assume that child procs wont need this. Bang the door if they do.